src/stdlib/WebSharper.JavaScript/Dom.fs

```diff
@@ -895,6 +1180,24 @@ module Interfaces =
                     Node *
                     T<string>?namespaceURI *
                     T<string>?qualifiedName ^-> Node
+
+                "close" => T<unit -> unit>
+                "open" => T<unit -> unit>
+                "write" => T<string -> unit>
+                "writeln" => T<string -> unit>
+
+                "execCommand" => T<string * bool * string -> bool>

```

Comment:
Methods like this should be declared as `T<string> * T<bool> * T<string> ^-> T<bool>`, to make sure that it doesn't mistakenly get translated to a tupled argument.